server/server.js

```diff
@@ -1,20 +1,85 @@
 import Database from "better-sqlite3";
 import { serve } from "@hono/node-server";
 import { Hono } from "hono";
+import { HTTPException } from "hono/http-exception";
+import bcrypt from "bcrypt";
 import queries from "./queries.js";
+const SALT_ROUNDS = 12;
```

Comment:
```suggestion

const SALT_ROUNDS = 12;
```

スタイル的に `import` から一行開けた方がみやすそう！